OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ssh/AuthenticationData.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2015 Dominik Schürmann <dominik@dominikschuermann.de>
+ * Copyright (C) 2017 Vincent Breitmoser <v.breitmoser@mugenguild.com>
+ * Copyright (C) 2017 Andreas-Christian Hagau <dev@hagau.se>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.sufficientlysecure.keychain.ssh;
```

Comment:
AuthenticationOperation should be moved to .operations
AuthenticationResult should be moved to .operations.results
Where shall this and AuthenticationParcel go?

Reply:
Hmm, not really sure, but I think it's not that important right now. You can leave it in your ssh package.